policy/modules/kernel/files.if

```diff
@@ -434,6 +434,24 @@ interface(`files_tmpfs_file',`
 	typeattribute $1 tmpfsfile;
 ')
 
+########################################
+## <summary>
+##	dontaudit getattr on tmpfs files
+## </summary>
+## <param name="domain">
+##	<summary>
+##	Domain to not have stat on tmpfs files audited
+##	</summary>
+## </param>
+#
+interface(`files_dontaudit_tmpfs_file_getattr',`
```

Comment:
Please rename to `files_dontaudi_getattr_all_tmpfs_files`.

Reply:
done